packages/components/src/tabs/styles.ts

```diff
@@ -33,25 +33,38 @@ export const TabListWrapper = styled.div`
 		content: '';
 		position: absolute;
 		pointer-events: none;
+		transform-origin: top left;
 
 		// Windows high contrast mode.
 		outline: 2px solid transparent;
 		outline-offset: -1px;
 	}
-	&:not( [aria-orientation='vertical'] )::after {
-		bottom: 0;
-		left: var( --indicator-left );
-		width: var( --indicator-width );
-		height: 0;
-		border-bottom: var( --wp-admin-border-width-focus ) solid
-			${ COLORS.theme.accent };
+	&:not( [aria-orientation='vertical'] ) {
+		&::after {
+			bottom: 0;
+			height: 0;
+			width: 1px;
+			transform: translateX( calc( var( --indicator-left ) * 1px ) )
+				scaleX( var( --indicator-width ) );
+			border-bottom: var( --wp-admin-border-width-focus ) solid
+				${ COLORS.theme.accent };
+		}
+
+		&:dir( rtl )::after {
+			right: 0;
+			transform-origin: top right;
```

Comment:
I wonder if there is a logical version to `transform-origin` ? Maybe using numbers/percentages instead of keywords?

Reply:
That'd be cool but it doesn't seem to be the case. I think it'd need to be a `start` or `end` keyword.